data/mods/ssb/moves.ts

```diff
@@ -907,6 +907,16 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 				}
 				return 5;
 			},
+			onModifyMove(move, attacker, defender) {
+				if (!move.overrideOffensiveStat) return;
+				if (!['atk', 'spa'].includes(move.overrideOffensiveStat)) return;
+				if (move.overrideOffensivePokemon === 'target') {
+					if (!defender) return;
+					move.overrideOffensiveStat = defender.getStat('atk') > defender.getStat('spa') ? 'spa' : 'atk';
+				} else {
+					move.overrideOffensiveStat = attacker.getStat('atk') > attacker.getStat('spa') ? 'spa' : 'atk';
+				}
+			},
```

Comment:
```suggestion
			onModifyMove(move, source, target) {
				if (!move.overrideOffensiveStat) return;
				if (!['atk', 'spa'].includes(move.overrideOffensiveStat)) return;
				const attacker = move.overrideOffensivePokemon === 'target' ? target : source;
				if (!attacker) return;
				move.overrideOffensiveStat = attacker.getStat('atk') > attacker.getStat('spa') ? 'spa' : 'atk';
			},
```

Is `if (!move.overrideOffensiveStat) return;` necessary? It seems wrong to me.

Reply:
That is a good rewrite. Makes sense.
It is necessary but it shouldn't be returning like that. That is an issue. I will fix that. It should just set it to atk or spa based on the move's catagory.